tests/event/event_simulation_tester.py

```diff
@@ -0,0 +1,126 @@
+#  Copyright 2023 Hathor Labs
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+from json import JSONDecodeError
+from typing import Any, Iterable
+from unittest.mock import Mock
+
+import pytest
+from twisted.internet.testing import StringTransport
+
+from hathor.event.storage import EventMemoryStorage, EventRocksDBStorage, EventStorage
+from hathor.event.websocket import EventWebsocketFactory, EventWebsocketProtocol
+from hathor.event.websocket.request import Request
+from hathor.event.websocket.response import EventResponse, InvalidRequestResponse
+from hathor.p2p.peer_id import PeerId
+from hathor.storage import RocksDBStorage
+from hathor.transaction.storage import TransactionStorage
+from hathor.transaction.util import unpack, unpack_len
+from hathor.util import json_loadb
+from tests.simulation.base import SimulatorTestCase
+from tests.utils import HAS_ROCKSDB
+
+
+class BaseEventSimulationTester(SimulatorTestCase):
+    seed_config = 6946502462188444706
```

Comment:
Why are we fixing a seed? Shouldn't we let random tests run?

Reply:
Removed in https://github.com/HathorNetwork/hathor-core/pull/647/commits/99cf418bb73591c67e42fd0520e7cd2ac94288e8

Reply:
Ok. Thanks!